symplyphysics/laws/kinematics/kinetic_energy_of_body_from_mass_and_velocity.py

```diff
@@ -0,0 +1,22 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+# Kinetic energy of body EK = (m * U^2) / 2
+# where :
```

Comment:
Remove space before colon. Make sure the description is formatted more or less same way as other laws.